pkg/autoscaler/statforwarder/processor.go

```diff
@@ -33,9 +33,40 @@ import (
 // accessed by IP address directly due to the network mesh.
 const establishTimeout = 500 * time.Millisecond
 
-// bucketProcessor includes the information about how to process
-// the StatMessage owned by a bucket.
-type bucketProcessor struct {
+type bucketProcessor interface {
+	process(asmetrics.StatMessage) error
+	is(holder string) bool
```

Comment:
nit: Add a comment for this fun? I could;n get what this func does only from its name.